edge/pkg/apis/dmi/services.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dmi
+
+import (
+	dmiapi "github.com/kubeedge/kubeedge/edge/pkg/apis/dmi/v1"
+)
+
+// DeviceManagerService defines the public APIS for remote device management.
+// The server is implemented by the module of device manager in edgecore
+// and the client is implemented by the device mapper for upstreaming.
+type DeviceManagerService interface {
```

Comment:
For DeviceManagerService: server is edgecore, client is mapper?
For DeviceMapperService: server is mapper, client is edgecore?

Reply:
yes. i know the current name may cause some confusion. but i have not yet thought of a more proper name.